app/src/common/utils/routingUtils.js

```diff
@@ -32,3 +32,5 @@ export const mergeQuery = (oldQuery, paramsToMerge) => {
     {},
   );
 };
+
+export const redirectAction = (linkAction) => linkAction;
```

Comment:
I don't think it's necessary since this function does nothing.

Reply:
It just wraps linkAction into action creator function. I can replace it by the same construction in @connect for each component like this:
```@connect(
null, 
{
 redirect: (linkAction) =>(linkAction)
}```
or
```@connect(
null,
(dispatch) => {
 redirect: (linkAction) => dispatch(linkAction)
}```
or I can write navigate action creator for each link.

Reply:
`@connect( null, { navigate: (linkAction) =>(linkAction) })`

Reply:
Fixed.